cpp/CMakeLists.txt

```diff
@@ -174,127 +174,312 @@ set(CUGRAPH_SOURCES
     src/detail/collect_local_vertex_values.cu
```

Comment:
This file also explicitly instantiates based on vertex_t, value_t, and multi_gpu, should I create sg32, sg64, mg32, and mg64?